installer_win/omsupply-postgres-build.bat

```diff
@@ -0,0 +1,3 @@
+@ECHO ##### Building omsupply for the postgres #####
+SET installerWorkspace=C:\Program Files (x86)\Jenkins\jobs\omSupplyMain - installers\workspace\omSupply
+cd .. && cd "server" && cargo clean && cd .. && cd "client" && yarn install --force && yarn build && cd .. && cd "server" && cargo build --release --features postgres && cd .. && @ECHO ##### Copying build artifacts to the installer folder ##### && copy "server\target\release\remote_server.exe" "%installerWorkspace%\omSupply Web Server\omSupply-postgres.exe" && copy "server\target\release\remote_server.exe" "%installerWorkspace%\omSupply Desktop Server\server\omSupply-postgres.exe"

```

Comment:
the command would be slightly neater if you just went into the client dir first, built the client then went ../server and ran all the server commands.

e.g.
```suggestion
cd .. && cd client && yarn install --force && yarn build && cd .. && cd server && cargo clean && cargo build --release --features postgres && cd .. && @ECHO ##### Copying build artifacts to the installer folder ##### && copy "server\target\release\remote_server.exe" "%installerWorkspace%\omSupply Web Server\omSupply-postgres.exe" && copy "server\target\release\remote_server.exe" "%installerWorkspace%\omSupply Desktop Server\server\omSupply-postgres.exe"
```